app/controllers/api/v1/api_controller.rb

```diff
@@ -10,7 +10,7 @@ class ApiController < ApplicationController
 
       def user_not_authorized
         flash[:warning] = "You have met a terrible fate, haven't you?"
-        redirect_to api_v1_book_index_path
+        render json: "Autorization failure #{current_user.id}"
```

Comment:
🤔 I recently constat I completly let this pass... You do the rigth thing here 👏 
Only one thing, give this the status `:unauthorized`